msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -186,6 +186,13 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 		<RemoveDir Directories="$(IntermediateOutputPath)" />
 	</Target>
 
+	<Target Name="_AddExtraReferences" BeforeTargets="ResolveAssemblyReferences" Condition="'$(DisableAutomaticSystemDrawingCommonReference)' == ''">
```

Comment:
I think `DisableAutomaticSystemDrawingCommonReference` should have a generic name to match the target name like `DisableExtraReferences` or `DisableAutomaticExtraReferences`.

Besides that, since it's a boolean property it'd be better give it a default value if it's empty (set it to false) or check if it's != 'true', because if someone sets it to false for any reason the target will be skipped when it shouldn't.

Reply:
@emaf ok, I've changed it to use `DisableExtraReferences` and check for (not) `true` instead of empty string.